packages/bsky/tests/auto-moderator/fuzzy-matcher.test.ts

```diff
@@ -37,7 +37,7 @@ describe('fuzzy matcher', () => {
   const getAllReports = () => {
     return network.bsky.ctx.db
       .getPrimary()
-      .db.selectFrom('moderation_report')
+      .db.selectFrom('moderation_event')
```

Comment:
Can we make it so that this function `getAllReports()` returns only reports rather than all events?